elasticdl/python/worker/worker.py

```diff
@@ -651,22 +652,15 @@ def _process_minibatch(
             self._run_model_call_before_training(features)
         for _ in range(self._max_minibatch_retry_num):
             if task_type == elasticdl_pb2.EVALUATION:
-                if min_model_version == -1:
-                    if self._model_version < 0:
-                        self.get_model(0, elasticdl_pb2.MINIMUM)
-                elif self._model_version != min_model_version:
-                    self.get_model(min_model_version, elasticdl_pb2.FIXED)
+                self.get_model()
```

Comment:
No need to get_model for every minibatch if the task_type is EVALUATION, especially for evaluation_only job.

Reply:
Done